assets/js/googlesitekit/datastore/user/disconnect.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * core/user Data store: disconnect
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+
+// Actions
+const START_FETCH_DISCONNECT = 'START_FETCH_DISCONNECT';
+const FETCH_DISCONNECT = 'FETCH_DISCONNECT';
+const FINISH_DISCONNECT = 'FINISH_DISCONNET';
```

Comment:
Bit of a typo in the value but this is slightly off in the formatting as well.
```suggestion
const FINISH_FETCH_DISCONNECT = 'FINISH_FETCH_DISCONNECT';
```